From e8082f2c75a0893f6367c1a23b6f10df67ac5c81 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 13 Sep 2019 11:24:39 +0200
Subject: [PATCH 1/2] appdata: Add ODRS ratings

---
 data/rhythmbox.appdata.xml.in | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/data/rhythmbox.appdata.xml.in b/data/rhythmbox.appdata.xml.in
index 0c79e659f..4dc7d379a 100644
--- a/data/rhythmbox.appdata.xml.in
+++ b/data/rhythmbox.appdata.xml.in
@@ -31,4 +31,33 @@
   <update_contact>jmatthew@gnome.org</update_contact>
   <project_group>GNOME</project_group>
   <translation type="gettext">rhythmbox</translation>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
 </component>
-- 
2.21.0


From 4e13a0997ad98d3f476d4d82adfb40f52577b50c Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 13 Sep 2019 11:29:24 +0200
Subject: [PATCH 2/2] appdata: Add 3.4.3 release information

---
 data/rhythmbox.appdata.xml.in | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/data/rhythmbox.appdata.xml.in b/data/rhythmbox.appdata.xml.in
index 4dc7d379a..0d0c1b7c2 100644
--- a/data/rhythmbox.appdata.xml.in
+++ b/data/rhythmbox.appdata.xml.in
@@ -31,6 +31,17 @@
   <update_contact>jmatthew@gnome.org</update_contact>
   <project_group>GNOME</project_group>
   <translation type="gettext">rhythmbox</translation>
+  <releases>
+    <release version="3.4.3" date="2018-12-31">
+      <description>
+        <p>
+          This new version fixes a number of bugs. The Android plugin now
+          works better with Samsung and Android 9 devices, and the status bar
+          was replaced with a floating bar.
+        </p>
+      </description>
+    </release>
+  </releases>
   <content_rating type="oars-1.1">
     <content_attribute id="violence-cartoon">none</content_attribute>
     <content_attribute id="violence-fantasy">none</content_attribute>
-- 
2.21.0

